platform/viewer/cypress/support/aliases.js

```diff
@@ -52,3 +52,28 @@ export function initVTKToolsAliases() {
   cy.get('.ohif-check-label').as('modeCheckbox');
   cy.get('.btn-group > .toolbar-button').as('layoutBtn');
 }
+
+//Creating aliases for Study List page elements on Desktop experience
+export function initStudyListAliasesOnDesktop() {
+  cy.get('.study-count').as('studyCount');
+  cy.get('#filter-patientName').as('patientName');
+  cy.get('#filter-patientId').as('MRN');
+  cy.get('#filter-accessionNumber').as('accessionNumber');
+  cy.get('#start-date').as('studyListStartDate');
+  cy.get('#end-date').as('studyListEndDate');
+  cy.get('#filter-modalities').as('modalities');
+  cy.get('#filter-studyDescription').as('studyDescription');
+  cy.get('[data-cy="study-list-results"] > tr').as('searchResult');
+}
+
+//Creating aliases for Study List page elements on Tablet experience
+export function initStudyListAliasesOnTablet() {
+  cy.get('.study-count').as('studyCount');
```

Comment:
You could use these methods to resize to the correct width before setting aliases?

Reply:
Yes, the resizing is being done before calling this function.